### ISSUE TITLE
Cloudformation ignores conditions

### ISSUE THREAD

**Issue by scottiemc7** [localstack/localstack#3654]
# Type of request: This is a ...

[X] bug report
[ ] feature request

# Detailed description
I swear this worked before, so I apologize in advance if this is actually a feature request. It appears that CloudFormation does not consider conditions when creating a stack.

## Expected behavior
Conditions can be used to selectively create resources.

## Actual behavior
Conditions are ignored and all resources in the template are created.

# Steps to reproduce
Create a stack with conditions. In this example stack, the default is for the parameter `Create` to be set to 'False' which will cause the condition to resolve to `false`. It also creates all resources if you explicitly set the `Create` parameter to 'False'.
`awslocal cloudformation create-stack --stack-name "TestConditions" --template-body file://conditions.template --capabilities CAPABILITY_IAM`
or
`awslocal cloudformation create-stack --stack-name "TestConditions" --template-body file://conditions.template --capabilities CAPABILITY_IAM --parameters ParameterKey=Create,ParameterValue='False'`

Check and see that resources with condition were created
`awslocal iam list-roles` ...

#### Template
[conditions.zip](https://github.com/localstack/localstack/files/6053160/conditions.zip)

## Command used to start LocalStack
`docker-compose up`

```
version: '2.1'

services:

localstack:
image: localstack/localstack:latest
container_name: localstack_main
ports:

"4566:4566"
"4567-4584:4567-4584"
"${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}"
environment:
SERVICES=lambda,cloudformation,iam,logs,events,secretsmanager,ssm,sqs,s3,apigateway,stepfunctions,ec2,sns
HOSTNAME=localhost
START_WEB=0
DEBUG=1
LAMBDA_EXECUTOR=docker
LAMBDA_REMOTE_DOCKER=true
DATA_DIR=/tmp/localstack/data
volumes:
${LOCALSTACK_DIR}/startup:/docker-entrypoint-initaws.d
"//var/run/docker.sock:/var/run/docker.sock"
```

**Comment by scottiemc7** [comment]
Looks like this did work recently #2083

**Comment by whummer** [comment]
Thanks for reporting @scottiemc7 . This should be fixed in 9edfa0c - can you please pull the `latest` Docker image and give it another try? Please let us know if the problem persists. Thanks

**Comment by scottiemc7** [comment]
This seems to work again. Thank you!
